Sources/SwifterSwift/UIKit/UIAlertControllerExtensions.swift

```diff
@@ -94,7 +94,7 @@ public extension UIAlertController {
             view.tintColor = color
         }
     }
-
+    
```

Comment:
@NickNguyeniOS14 here's an example of some spacing that still needs to be fixed. You can see all of them inside the diff of `UIAlertControllerExtensions.swift` and `UIAlertControllerExtensionsTests.swift`

Reply:
@SD10 @NickNguyeniOS14 Running `swiftlint autocorrect` should fix them all :)

Reply:
You can install swiftlint with `brew install swiftlint` or see instructions [here](https://github.com/realm/SwiftLint#installation)